crates/worker/src/partition/state_machine/effect_interpreter.rs

```diff
@@ -167,6 +169,7 @@ where
         let Self { txn, collector } = self;
 
         txn.commit().await?;
+        counter!(PARTITION_STORAGE_TX_COMMITTED).increment(1);
```

Comment:
When rebasing on `main`, this increment should no longer be needed.

Reply:
Care to elaborate on why?